views-vues/agile/en/agile-use-when.html

```diff
@@ -1,133 +0,0 @@
----
```

Comment:
Is there a reason this is deleted?

Reply:
Some local changes for efficiency accidentally ended up in a commit.
Its been reverted, and the change recommited.